operations/research-and-development/product/technical-writing-team-handbook/README.md

```diff
@@ -46,34 +46,36 @@ Other areas of documentation include:
 
 Please reach out to the [Documentation Working Group](https://community.mattermost.com/private-core/channels/dwg-documentation-working-group) for any questions.
 
-### Contributing to documentation
+## How to contribute to the documentation
 
-The basic outline for getting started with contributions is provided in the [README](https://github.com/mattermost/docs/blob/master/README.md) of the docs repo. If you have write access to the repo, you can create a branch off master and work on that.
+Our documentation is open to contributions from anyone. The basic outline for getting started with contributions is provided in the [README](https://github.com/mattermost/docs/blob/master/README.md) of the docs repo. If you have write access to the repo, you can create a branch off master and work on that.
```

Comment:
I wonder if we want to make this even more inclusive by including details on using the Edit on GitHub option (which auto-creates a fork if the user doesn't have appropriate permissions)?

Reply:
The link in the next sentence does provide that information with more context but we can definitely add it here as well.